components/sysutils/testdisk/Makefile

```diff
@@ -21,16 +21,15 @@
 # Copyright (c) 2016 by Jim Klimov. All rights reserved.
 # Copyright (c) 2019 Tim Mooney <Timothy.V.Mooney@gmail.com>
 #
-include ../../make-rules/shared-macros.mk
+include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		testdisk
 COMPONENT_VERSION=	7.0
-COMPONENT_REVISION=	1
```

Comment:
Shouldn't it be bumped to `2` instead or removed?

Reply:
You are right. I found this problem by myself but you have been faster :)
I removed it when trying to update to 7.1.
Regards